### PR TITLE
Get Raw Transaction Endpoint Implementation

### PR DESCRIPTION
**Pull request by noumanahmads**
## Description

Get Raw transaction from `tx_id`.
New endpoint `/:tx_id/raw` added. 

For details refer to issue #428 

## Type of Change
- [x] New feature
- [ ] Bug fix
- [ ] API reference/documentation update
- [ ] Other

## Does this introduce a breaking change?
No

## Are documentation updates required?
Yes, this endpoint should be documented somewhere. 

## Testing information
npm run test

## Checklist
- [x] Code is commented where needed
- [x] Unit test coverage for new or modified code paths
- [x] `npm run test` passes
- [ ] Changelog is updated
- [x] @zone117x for review

### CODE REVIEW

File: src/datastore/postgres-store.ts [hunk 1]
```diff
@@ -306,6 +306,10 @@ interface TransferQueryResult {
   amount: string;
 }
 
+export interface RawTxQueryResult {
+  raw_tx: string;
```
Comment (zone117x) [inline src/datastore/postgres-store.ts:310]: This is stored as a `bytea` column type, which means this return value should be `raw_tx: Buffer`?

File: src/datastore/postgres-store.ts [hunk 2]
```diff
@@ -2936,6 +2940,28 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
...
+        ORDER BY canonical DESC, block_height DESC
+        LIMIT 1
+        `,
+        [hexToBuffer(txId)]
+      );
+      if (result.rowCount === 0) {
+        return { found: false } as const;
+      }
+      const queryResult: RawTxQueryResult = {
+        raw_tx: result.rows[0].raw_tx.toString(),
```
Comment (zone117x) [inline src/datastore/postgres-store.ts:2959]: `bufferToHexPrefixString` should be used to convert the `raw_tx` Buffer to a hex string.

File: docs/api/transaction/get-raw-transaction.example.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{ 
+  "raw_tx": "test-raw-tx"
```
Comment (zone117x) [inline docs/api/transaction/get-raw-transaction.example.json:2]: `"raw_tx: "0x1234"` would be more inline with the other examples

File: docs/api/transaction/get-raw-transaction.schema.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "GET raw transaction",
+  "title": "GetRawTransactionResult",
+  "type": "object",
+  "required": ["raw_tx"],
+  "properties": {
+    "raw_tx": {
+      "type":"string",
+      "description": "raw tx details"
```
Comment (zone117x) [inline docs/api/transaction/get-raw-transaction.schema.json:10]: A more accurate description would be something like "A hex encoded serialized transaction"

File: docs/api/transaction/get-raw-transaction.example.json (hunk 3 above)
Comment (noumanahmads) [reply docs/api/transaction/get-raw-transaction.example.json]: done.

File: docs/api/transaction/get-raw-transaction.schema.json (hunk 4 above)
Comment (noumanahmads) [reply docs/api/transaction/get-raw-transaction.schema.json]: Updated description.

**Review by zone117x — APPROVED** [review]
LGTM. Thanks @noumanahmads!

**Pull request merged**